python/pfs/drp/stella/measureCrosstalk.py

```diff
@@ -0,0 +1,345 @@
+        self.isr.doSaturationInterpolation = False
+        self.isr.maskListToInterpolate = []
+        # Background subtraction
+        self.subtractBackground.statisticsProperty = "MEDIAN"
+        self.subtractBackground.binSize = 512
+        self.subtractBackground.useApprox = False
+        MaskX.addMaskPlane("FIBERTRACE")
+        self.subtractBackground.ignoredPixelMask.append("FIBERTRACE")
+        self.buildFiberTraces.doBlindFind = True  # Traces shift around in a dithered flat sequence
+
+
+class MeasureCrosstalkRunner(TaskRunner):
+    def __call__(self, *args, **kwargs):
+        """Remove the dataRef from the result
+
+        Otherwise, running threaded yields:
+
+            sqlite3.ProgrammingError: SQLite objects created in a thread can
+            only be used in that same thread.
+        """
+        result = TaskRunner.__call__(self, *args, **kwargs)
+        if hasattr(result, "dataRef"):
+            del result.dataRef
+        return result
+
+
+class MeasureCrosstalkTask(CmdLineTask):
+    """Measure intra-CCD crosstalk
+
+    This Task behaves in a scatter-gather fashion:
+    * Scatter: get ratios for each CCD.
+    * Gather: combine ratios to produce crosstalk coefficients.
+    """
+    ConfigClass = MeasureCrosstalkConfig
+    RunnerClass = MeasureCrosstalkRunner
+    _DefaultName = "measureCrosstalk"
+
+    def __init__(self, *args, **kwargs):
+        CmdLineTask.__init__(self, *args, **kwargs)
+        self.makeSubtask("isr")
+        self.makeSubtask("subtractBackground")
+        self.makeSubtask("buildFiberTraces")
+
+    @classmethod
+    def _makeArgumentParser(cls):
+        parser = super(MeasureCrosstalkTask, cls)._makeArgumentParser()
+        parser.add_argument("--crosstalkName",
+                            help="Name for this set of crosstalk coefficients", default="Unknown")
+        parser.add_argument("--outputFileName",
+                            help="Name of yaml file to which to write crosstalk coefficients")
+        parser.add_argument("--dump-ratios", dest="dumpRatios",
+                            help="Name of pickle file to which to write crosstalk ratios")
+        return parser
+
+    @classmethod
+    def parseAndRun(cls, *args, **kwargs):
+        """Implement scatter/gather
+
+        Returns
+        -------
+        coeff : `numpy.ndarray`
+            Crosstalk coefficients.
+        coeffErr : `numpy.ndarray`
+            Crosstalk coefficient errors.
+        coeffNum : `numpy.ndarray`
+            Number of pixels used for crosstalk measurement.
+        """
+        kwargs["doReturnResults"] = True
+        results = super(MeasureCrosstalkTask, cls).parseAndRun(*args, **kwargs)
+        task = cls(config=results.parsedCmd.config, log=results.parsedCmd.log)
+        resultList = [rr.result for rr in results.resultList]
+        if results.parsedCmd.dumpRatios:
+            import pickle
+            pickle.dump(resultList, open(results.parsedCmd.dumpRatios, "wb"))
```

Comment:
Shouldn't docString mention that the pickle file may be dumped in this method (as a by-product)?